edx_proctoring/templates/proctored_exam/instructions.html

```diff
@@ -38,8 +38,44 @@
         You will be asked to verify your identity before you begin the exam. Make sure you have valid photo identification, such as a driver's license or passport, before you continue.
       {% endblocktrans %}
     </p>
+    <p>
+        {% blocktrans %}
+          When you have finished setting up the proctoring application, you may start the exam.
```

Comment:
Can we make this step 3 as Chris suggests, and change the text to:
"3. When you have finished setting up proctoring, start the exam."